tests/hooks/mock/seminarSeriesMock.ts

```diff
@@ -0,0 +1,84 @@
+export const mock: any[] = [
+  {
+    Name: "delectus eaque consectetur",
+    "Event Start Date": "2022-07-29",
+    "Type of Engagement": ["Seminar Series"],
+    "Event Website": "http://example.com",
+    "Event City/State": "YouTube",
+    Region: ["Online"],
+    "Image / Icon": [
+      {
+        id: "18f3a44d-cae2-4e12-9403-25dbdc5b90a2",
+        width: 295,
+        height: 295,
+        url: "https://example.com",
+        filename: "french_e_services_digital.docx",
+        size: 60078,
+        type: "image/jpeg",
+        thumbnails: {
+          small: {
+            url: "https://example.com",
+            width: 36,
+            height: 36,
+          },
+          large: {
+            url: "https://example.com",
+            width: 295,
+            height: 295,
+          },
+          full: {
+            url: "https://example.com",
+            width: 3000,
+            height: 3000,
+          },
+        },
+      },
+    ],
+    Institution: "Okuneva Group",
+    "Speaker (S.S.)": "Lela Bahringer",
+    "Add to Seminar Series Site": true,
+  },
+  {
+    Name: "Ea commodi earum id.",
+    "Event Start Date": "2023-06-09",
+    "Event Start Date Time": "2023-06-09 14:00:00",
+    "Type of Engagement": ["Seminar Series"],
+    "Event City/State": "YouTube",
+    "Event Abstract": "Qui facere assumenda ducimus.",
+    "Add to Event Site": true,
+    "Event Website": "http://example.com",
+    "Image / Icon": [
+      {
+        id: "att8eF6mVuKzbZxk0",
+        width: 604,
+        height: 684,
+        url: "http://dimwitted-claw.org",
+        filename: "intelligent_deposit.sass",
+        size: 400718,
+        type: "image/png",
+        thumbnails: {
+          small: {
+            url: "http://amused-microwave.org",
```

Comment:
It might be safer to replace all mock URLs with `https://example.com/` or `https://example.org/` (a path can be appended), since it's a safe reserved URL for testing.

Relevant documents:

- https://www.rfc-editor.org/rfc/rfc2606.html
- https://www.rfc-editor.org/rfc/rfc6761.html

Reply:
Done